components/Storyblok/SbLocalFooter.tsx

```diff
@@ -28,83 +27,81 @@ export type SbLocalFooterProps = {
 };
 
 const styles = {
-  linkGroup: cnb('list-none m-0 p-0 [&_a]:leading-snug [&_a]:font-normal [&_a]:text-digital-red [&_a:hover]:text-black [&_a:focus]:text-black'),
-  linkGroupHeading: cnb('text-20 font-serif tracking-normal'),
-}; 
+  linkGroup: 'list-none m-0 p-0 [&_a]:leading-snug [&_a]:font-normal [&_a:hover]:text-black [&_a:focus]:text-black',
+  linkGroupHeading: 'text-20',
+};
 
 export const SbLocalFooter = (props: SbLocalFooterProps) => (
-  <div {...storyblokEditable(props.blok)} className="pb-38 pt-34 md:pb-72 md:pt-58 bg-white">
-    <CenteredContainer>
```

Comment:
I didn't actually change the local footer much - I consolidated the outermost `div` wrapper and the old `<CenterContainer>` with `<Container>`, and replaced the old `Heading` component with the new `Heading` + some minor tweaks here and there. Please double check.